app/javascript/components/ingredients/base-form-component.vue

```diff
@@ -92,18 +82,20 @@
 
 export default {
   props: {
-    editMode: {
-      type: Boolean,
-      required: true },
-    ingredient: {
-      type: String,
-    },
+    editMode: { type: Boolean, required: true },
+    ingredient: { type: Object, default() {
```

Comment:
acá el default en vez de ser una función podría ser `default: {}` o no?

Reply:
Lo intenté pero me tiraba un error de que `default value for 'ingredient' prop must be a function`

Reply:
toda la razón, [acá](https://vuejs.org/v2/guide/components-props.html) lo explican en la documentación